app/faucet/templates/faucet_form.html

```diff
@@ -88,14 +88,6 @@ <h3>{% trans "Faucet Request" %} <span id="alpha">{% trans "Alpha" %}</span></h3
             <div id="success_container" class="faucet_response text-center">
               <h5 class="mt-5 mb-4">{% trans "Request Received" %}</h5>
               {% include 'svgs/success.svg' %}
-              <div class="fail_message mt-4">
-                <p class="mb-0">{% trans "Your request for a faucet distribution has been received." %}</p>
-                <p class="mb-3">{% trans "The Gitcoin team will review your request shortly." %}</p>
-                <p class="mb-0">{% trans "While you wait, why not create an *free* Gitcoin ENS name?" %}</p>
```

Comment:
Should we just remove the ENS portion of the message and leave the rest?

Reply:
addressed !